### PR TITLE
youtubedr 2.2.1 (new formula)

### PR DESCRIPTION
**Pull request by kkc**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/youtubedr.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      -X main.date=#{build_time}
+    ]
+
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build", "-ldflags", ldflags.join(" "), *std_go_args, "./cmd/youtubedr"
+  end
+
+  test do
+    output = pipe_output("#{bin}/youtubedr version")[0..17]
+    assert_match(/Version:     2.2.1/, output)
```
Comment (corny) [inline Formula/youtubedr.rb:27]: better usethe version variable:
```ruby
assert_match(/Version:\s+#{Regexp.escape(version)}/, output)
```

Comment (kkc) [reply Formula/youtubedr.rb]: fixed

**Comment by dtrodrigues** [comment]
Why is this including `bottle :unneeded`?

File: Formula/youtubedr.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+class Youtubedr < Formula
+  desc "Download Youtube Video in Golang"
+  homepage "https://github.com/kkdai/youtube"
+  url "https://github.com/kkdai/youtube/archive/v2.2.1.tar.gz"
+  sha256 "e0e298aa4d589813b635bc09e9b50b220f248312ec5893a381bacf70a624a9e1"
+  license "MIT"
+
+  bottle :unneeded
```
Comment (SMillerDev) [inline Formula/youtubedr.rb:8]: It does need a bottle

File: Formula/youtubedr.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      -X main.date=#{build_time}
+    ]
+
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build", "-ldflags", ldflags.join(" "), *std_go_args, "./cmd/youtubedr"
+  end
+
+  test do
+    version_output = pipe_output("#{bin}/youtubedr version").split("\n")
+    assert_match(/Version:\s+#{Regexp.escape(version)}/, version_output[0])
```
Comment (SMillerDev) [inline Formula/youtubedr.rb:27]:
```suggestion
    assert_match "Version: #{version}", version_output[0]
```

File: Formula/youtubedr.rb (hunk 2 above)
Comment (kkc) [reply Formula/youtubedr.rb:8]: sure, I will figure it out how to add a bottle

File: Formula/youtubedr.rb (hunk 3 above)
Comment (kkc) [reply Formula/youtubedr.rb:27]: thx a lot !

File: Formula/youtubedr.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/youtubedr.rb:8]: You don't need to do anything, just don't specify that it doesn't need a bottle.

https://docs.brew.sh/Bottles#creation
> Bottles are created using the Brew Test Bot, usually when people submit pull requests to Homebrew. The bottle do block is updated by maintainers when they merge a pull request.

**Review by SMillerDev — APPROVED** [review]
Thanks @kkc ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**